README.md

```diff
@@ -1,5 +1,5 @@
 ## License
-This template is licensed to Customer subject to the terms of the license agreement between Domino and the Customer on file.
+This template is licensed under Apache 2.0 and contains the following open source components: Langchain ([MIT](https://github.com/langchain-ai/langchain/blob/master/LICENSE), Pinecone ([EULA](https://www.pinecone.io/thin-client-user-agreement/), [OpenAI](https://openai.com/policies/terms-of-use), and [FAISS](https://github.com/facebookresearch/faiss/blob/main/LICENSE)).
```

Comment:
<img width="659" alt="Screen Shot 2023-12-20 at 1 24 13 PM" src="https://github.com/dominodatalab/reference-project-customqa/assets/6508755/16847a2d-c617-409b-a9cb-35250ffbdcf8">
 the parenthesis look unbalanced

Reply:
Good catch, fixed @niole .